demo/demoproject/fixtures/meter-readings.json

```diff
@@ -0,0 +1,444 @@
+[
+{
```

Comment:
Can you export only the fixtures for `demoproject.meter` ./manage.py dumpdata demo.meter ?

Reply:
Oops, I missed this also, sorry about that.